pan3d/dataset_builder.py

```diff
@@ -258,7 +258,7 @@ def mesh(
 
     def _call_catalog_function(self, catalog_name, function_name, **kwargs):
```

Comment:
I don't see `self` being used here. Should that be in `pan3d/catalogs/__init__.py` which also seems to be missing.

Reply:
Sure, changed in [0cc5c57](https://github.com/Kitware/pan3d/pull/69/commits/0cc5c578b33a48fbe4968f9f147e61f81cb0152f)